modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -23,6 +23,50 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 8
+
+Infinite Scale 2.0.0 Beta 8 includes a huge number of smaller bug fixes and round-offs all around. It introduces a more efficient and scalable way to store share information and improves the performance of ownCloud Web in many areas very noticeable. Furthermore it is now possible to set user quotas in the UI and administrators can now delete orphaned Spaces.
+
+IMPORTANT: When updating from Infinite Scale Beta 7 or lower to Beta 8 it is necessary to run two migration steps from the previous to the new share manager **before** starting the system again regularly.
+
+**Migration for User/Group Shares**
+
+`OCIS_LOG_LEVEL="debug" OCIS_LOG_PRETTY="true" OCIS_LOG_COLOR="true" ocis migrate shares --from cs3 --to jsoncs3`
+
+**Migration for Public Links**
+
+`OCIS_LOG_LEVEL="debug" OCIS_LOG_PRETTY="true" OCIS_LOG_COLOR="true" ocis migrate publicshares --from cs3 --to jsoncs3`
+
+If you have set up your system with Docker, you can just use the same syntax as with https://doc.owncloud.com/ocis/next/deployment/container/container-setup.html#first-time-start[ocis init]:
+
+```
+docker run --rm -it \
+    --mount type=bind,source=$PWD/ocis/ocis-config,target=/etc/ocis \
+    owncloud/ocis migrate shares
+```
+
+The most prominent changes in **Infinite Scale 2.0.0 beta8** and ownCloud Web 5.7.0 comprise:
+
+* A new share manager implementation has been added that will store share information in a more efficient and scalable way. https://github.com/owncloud/ocis/issues/4431[ocis#4431] https://github.com/cs3org/reva/pull/3199[reva#3199]
+* The performance of loading the file list has been drastically improved. https://github.com/owncloud/web/issues/7038[web#7038]
```

Comment:
That was not part of this beta. We only limited the number of items per page for pagination to `100` or `500`, the options `1000` and `All` have been removed.

Reply:
Ah, then the changelog was a little misleading.